### PR TITLE
Add text to speech functionality

### PR DESCRIPTION
**Pull request by jwang281**
Add a button after each message that will read the message for you

### CODE REVIEW

File: src/main/webapp/WEB-INF/view/chat.jsp [hunk 1]
```diff
@@ -96,9 +108,10 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
         String renderedMessage = renderer.render(document);
         
         String result = EmojiParser.parseToUnicode(renderedMessage);	
-		String resultDecimal = EmojiParser.parseToHtmlDecimal(result);	  	
+		String resultDecimal = EmojiParser.parseToHtmlDecimal(result);	  
+		String text = EmojiParser.parseToAliases(Jsoup.parse(renderedMessage).text());
     %>
-      <li><strong><%= author %>:</strong> <%= resultDecimal %> </li>
+      <li><strong><%= author %>:</strong> <%= resultDecimal %> <button onclick="playMsg('<%= text %>')">&#x1F50A;</button></li>
```
Comment (ghost) [inline src/main/webapp/WEB-INF/view/chat.jsp]: Great I wonder if there will be merge conflicts with `chat.jsp`. Is this supported in every browser?

File: src/main/webapp/WEB-INF/view/chat.jsp [hunk 2]
```diff
@@ -56,6 +59,15 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       var chatDiv = document.getElementById('chat');
       chatDiv.scrollTop = chatDiv.scrollHeight;
     };
+	function playMsg(m) {
+	  console.log('inside func');
```
Comment (KevinWorkman) [inline src/main/webapp/WEB-INF/view/chat.jsp]: You probably don't need these kinds of print statements. They're useful for debugging, but probably not for the real product. It's not a big deal though so it's up to you!

**Review by KevinWorkman — APPROVED** [review]
Nice work Ivy! This looks really promising.

**Review by scoheleach — APPROVED** [review]
awesome!!

**Review by ghost — APPROVED** [review]
Looks very interesting. I think in order to solve the merge conflict just update your branch with master and then merge.

**Pull request merged**